app/config/config.test.json

```diff
@@ -21,5 +21,9 @@
 		}
 	},
 	"operator-email": "email@operatorsownmailserver.com",
-	"operator-displayname": "Friendly Operator"
+	"operator-displayname": "Friendly Operator",
+	"mail-recipients": {
+		"add-subscription": "email@operatorsownmailserver.com",
```

Comment:
should these not be different from `operator-email` so you can catch them getting mixed up in the code?